apps/cli/src/commands/doctor.ts

```diff
@@ -11,102 +11,104 @@ export default class DoctorCommand extends Command {
     private static MINIMUM_DOCKER_VERSION = "23.0.0"; // Replace with our minimum required Docker version
     private static MINIMUM_DOCKER_COMPOSE_VERSION = "2.21.0"; // Replace with our minimum required Docker Compose version
 
-    private static async checkDockerVersion(): Promise<string> {
-        const { stdout } = await execa("docker", [
-            "version",
-            "--format",
-            "{{json .Client.Version}}",
-        ]);
-        return JSON.parse(stdout);
-    }
-
-    private static async checkDockerComposeVersion(): Promise<string> {
-        const { stdout } = await execa("docker", [
-            "compose",
-            "version",
-            "--short",
-        ]);
-        return stdout;
-    }
-
-    private static isDockerVersionValid(version: string): boolean {
-        return semver.gte(version, DoctorCommand.MINIMUM_DOCKER_VERSION);
-    }
+    private async checkDocker(): Promise<true | never> {
```

Comment:
@tuler I'd like your opinion on this return signature `Promise<true | neve>`.

My goal was to make proper error treatment, but I see the `true` return is a little weird, since I neve use it.

Reply:
Actually, I'm using the `true` return in the last commit, where I check if it's ok before checking the `compose` and `buildx` plugin commands.